<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>732</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="29" column="1">
    <widget class="QDialogButtonBox" name="terminer">
     <property name="font">
      <font>
       <family>Tw Cen MT</family>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="29" column="0">
    <widget class="QProgressBar" name="progressBar">
     <property name="font">
      <font>
       <family>Tw Cen MT</family>
      </font>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="3">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>Tw Cen MT</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Pour la partie temporelle de mon analyse, je souhaite utiliser...</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="horizontalSpacing">
        <number>6</number>
       </property>
       <property name="verticalSpacing">
        <number>29</number>
       </property>
       <item row="0" column="1">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Tw Cen MT</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>...la date des relevés compris dans le fichier vecteur</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QRadioButton" name="radioVecteur">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>Tw Cen MT</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>...une intervalle que je précise</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="radioChoix">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_15">
           <property name="font">
            <font>
             <family>Tw Cen MT</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Nom du champ contenant la date :</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="nomChamp"/>
         </item>
        </layout>
       </item>
       <item row="1" column="2">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QDateEdit" name="dateDebut">
           <property name="font">
            <font>
             <family>Tw Cen MT</family>
            </font>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
           <property name="date">
            <date>
             <year>2025</year>
             <month>6</month>
             <day>1</day>
            </date>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateFin">
           <property name="font">
            <font>
             <family>Tw Cen MT</family>
            </font>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
           <property name="date">
            <date>
             <year>2025</year>
             <month>7</month>
             <day>1</day>
            </date>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>23</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>85</green>
             <blue>127</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>85</green>
             <blue>127</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>120</red>
             <green>120</green>
             <blue>120</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>85</green>
             <blue>127</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>85</green>
             <blue>127</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <family>Tw Cen MT</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Lancer mon analyse biodiversité</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>12</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Tw Cen MT</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Pour la partie spatiale de mon analyse, je souhaite utiliser un fichier vecteur...</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="horizontalSpacing">
          <number>6</number>
         </property>
         <property name="verticalSpacing">
          <number>11</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_16">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Tw Cen MT</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>...stocké en local : </string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QgsFileWidget" name="inputPoints">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_14">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Tw Cen MT</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>...contenu dans le projet en cours :</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QgsMapLayerComboBox" name="inputPoints_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="font">
        <font>
         <family>Tw Cen MT</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>NB : l'utilisateur est prié de vérifier que son fichier vecteur contient un champ id, un champ date si le relevé est ponctuel, un champ x et un champ y, et que son fichier vecteur soit bien constitué d'entités ponctuelles valides et référencé dans le bon système de coordonnées (Lambert 93, EPSG [2154])</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>6</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="25" column="0" colspan="3">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Tw Cen MT</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Fichier GPKG relatif à la zone d'étude et dont la table &quot;mesure&quot; a été complétée</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QgsFileWidget" name="inputGPKG"/>
     </item>
     <item>
      <spacer name="verticalSpacer_6">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>28</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <property name="spacing">
        <number>16</number>
       </property>
       <item>
        <widget class="QLabel" name="label_17">
         <property name="font">
          <font>
           <family>Tw Cen MT</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Nom de la couche finale : </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nomCouche"/>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="generer">
         <property name="font">
          <font>
           <family>Tw Cen MT</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Lancer l'analyse spatio-temporelle</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="11" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>22</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgsfilewidget.h</header>
  </customwidget>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>terminer</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>terminer</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
